src/gui/core/window_builder.cpp

```diff
@@ -70,11 +70,11 @@ builder_widget_lookup()
  * be tuned. This page will describe what can be tuned.
  *
  */
-twindow* build(CVideo& video, const twindow_builder::tresolution* definition)
+window* build(CVideo& video, const builder_window::window_resolution* definition)
```

Comment:
I'm not quite sure what you're suggesting here. `builder_window` is the class that builds a window; I swapped the word order only because that makes it match all the other classes that build various widgets, which are named things like `builder_grid`, `builder_listbox`, etc. Changing it to `build_window` would make it a verb... or are you suggesting to rename the function here?